code/game/machinery/computer/aifixer.dm

```diff
@@ -135,3 +131,10 @@
 			to_chat(user, "<span class='boldannounce'>ERROR</span>: Reconstruction in progress.")
 		else if(!occupant)
 			to_chat(user, "<span class='boldannounce'>ERROR</span>: Unable to locate artificial intelligence.")
+
+/obj/machinery/computer/aifixer/emp_act()
+	if(occupant)
+		restoring = 0
+		occupant.adjustBruteLoss(200)

```

Comment:
Tully is probably right here because well, when you EMP an AI it takes brute damage by default.